accessibility/house-style.md

```diff
@@ -0,0 +1,125 @@
+
+*Caveat:* it's possible to build a site that passes Pa11y's testing with flying colours, but is still completely inaccessible for real users. You MUST NOT rely on Pa11y as your sole safeguard for accessibility testing. 
+
+
+#### Other accessibility testing tools
+
+- Pa11y is based on HTMLCodesniffer by Squizlabs. They also have a [bookmarklet](https://squizlabs.github.io/HTML_CodeSniffer/) for quick tests. 
+- Google's Accessibility Developer Tools are available from the [Chrome Web Store](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en).
+- WebAim's [WAVE toolbar](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) for Chrome evaluates accessibility in place on the page.
+- Check colour contrast compliance with Jonathan Snook's [Colour Contrast Checker](https://snook.ca/technical/colour_contrast/colour.html#fg=33FF33,bg=333333).
+
+*Caveat:* None of these tools on their own will catch every error. Even by combining all of them, it's still possible to produce an inaccessible webpage. Terrill Thompson [compared several of the most popular tools](http://terrillthompson.com/blog/730) which should give you some idea of the scale of the problem. The Government Digital Service also [conducted their own tool audit](https://accessibility.blog.gov.uk/2017/02/24/what-we-found-when-we-tested-tools-on-the-worlds-least-accessible-webpage/). 
+
+You MUST NOT rely on accessibility tools as your sole safeguard for accessibility testing.
+
+
+#### Assistive technology
+
+You're encouraged to test your pages with assistive technology (AT). Many operating systems include some AT as standard, including screenreaders, magnifiers, or input remapping. 
+
+In OSX you can enable [VoiceOver](https://support.apple.com/kb/PH22549?locale=en_US) in System Preferences > Accessibility. You may also find navigating without an input device using [Dictation](https://support.apple.com/en-us/HT202584) instructive. Ensure your interfaces work with keyboard navigation. 
+
+*Caveat:* unless you're familiar with using AT, your experience won't be comparable to that of a habitual AT user. Beware of making inferences about what's "easier" for AT users based on your own preferences. 
+
+
+#### Testing with real users
+
+The best way to learn if your website works for people with disabilities is to get people with disabilities to test it for you. 
+
+You can arrange your own user testing, but it's better to engage a third party like the [Digital Accessibility Centre](http://www.digitalaccessibilitycentre.org/) to do this work for you. As professionals who specialise in accessibility testing and making recommendations, they're better at it than you. 
+
+
+#### Resources
+
+Udacity have published an [excellent free course on Web Accessibility](https://www.udacity.com/course/web-accessibility--ud891) with Google engineers Alice Boxhall and Rob Dodson. 
+
```

Comment:
Could you also link to some of GDS' excellent stuff? Namely: 
* https://gds.blog.gov.uk/2017/10/23/were-making-accessibility-clearer-and-easier/
* https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction

It's really _really_ good.